### PR TITLE
Changes to azure community meeting

### PR DESCRIPTION
**Pull request by anshulbehl**

### CODE REVIEW

File: meetings/azure.yaml [hunk 1]
```diff
@@ -2,16 +2,15 @@ project: Azure working group
 meeting_id: Azure
 project_url: https://github.com/ansible/community/wiki/azure
 agenda_url: https://github.com/ansible/community/issues?q=is:open+label:meeting_agenda+label:azure
-chair: Matt Davis (nitzmahone), Kylie Liang (kyliel)
+chair: Anshul Behl (anshulbehl), Jacob Jackson (thedoubl3j)
```
Comment (thedoubl3j) [inline meetings/azure.yaml]: All be it that I am fine with this for now, we will need to transition this to the respective members of the azure community, both azure devs and community members who wish to step forward and help run this.

Comment (anshulbehl) [reply meetings/azure.yaml]: couldn't change it to `?` as its not allowed when running `make`

Comment (thedoubl3j) [reply meetings/azure.yaml]: wfm, just need to make a note to make sure that it doesn't stay with us long ;)

File: meetings/ical/core-team.ics [hunk 2]
```diff
@@ -3,7 +3,7 @@ VERSION:2.0
 PRODID:-//yaml2ical agendas//EN
```
Comment (gundalow) [inline meetings/ical/core-team.ics]: Please revert this file.

File: meetings/ical/docs.ics [hunk 3]
```diff
@@ -3,7 +3,7 @@ VERSION:2.0
 PRODID:-//yaml2ical agendas//EN
```
Comment (gundalow) [inline meetings/ical/docs.ics]: Please revert this file.

File: meetings/ical/molecule.ics [hunk 4]
```diff
@@ -3,7 +3,7 @@ VERSION:2.0
 PRODID:-//yaml2ical agendas//EN
```
Comment (gundalow) [inline meetings/ical/molecule.ics]: Please revert this file.

File: meetings/ical/network.ics [hunk 5]
```diff
@@ -3,7 +3,7 @@ VERSION:2.0
 PRODID:-//yaml2ical agendas//EN
```
Comment (gundalow) [inline meetings/ical/network.ics]: Please revert this file.

File: meetings/ical/vmware.ics [hunk 6]
```diff
@@ -3,7 +3,7 @@ VERSION:2.0
 PRODID:-//yaml2ical agendas//EN
```
Comment (gundalow) [inline meetings/ical/vmware.ics]: Please revert this file.

File: meetings/ical/vmware.ics [hunk 7]
```diff
@@ -3,7 +3,7 @@ VERSION:2.0
...
 BEGIN:VEVENT
 SUMMARY:VMware working group
-DTSTART;VALUE=DATE-TIME:20181126T160000Z
+DTSTART;VALUE=DATE-TIME:20191028T160000Z
 DURATION:PT1H
 RRULE:FREQ=WEEKLY
 DESCRIPTION:Project:  VMware working group\nChair:  Abhijeet Kasurde (akas
```
Comment (gundalow) [inline meetings/ical/vmware.ics]: Please revert this file.

File: meetings/ical/windows.ics [hunk 8]
```diff
@@ -3,7 +3,7 @@ VERSION:2.0
 PRODID:-//yaml2ical agendas//EN
 BEGIN:VEVENT
 SUMMARY:Windows working group
```
Comment (gundalow) [inline meetings/ical/windows.ics]: Please revert this file.

File: ansible_community_meetings.ics [hunk 9]
```diff
@@ -75,7 +75,7 @@ LOCATION:#ansible-lockdown
 END:VEVENT
 BEGIN:VEVENT
 SUMMARY:Molecule working group
-DTSTART;VALUE=DATE-TIME:20190123T190000Z
+DTSTART;VALUE=DATE-TIME:20191030T190000Z
```
Comment (gundalow) [inline ansible_community_meetings.ics]: Please revert non-Azure changes to this file

File: meetings/README.md [hunk 10]
```diff
@@ -17,7 +17,7 @@ You can add this full list of meetings to your personal calendar by importing by
 
 * [15:00 UTC](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC):
   **[Security Automation Working Group](https://github.com/ansible/community/wiki/Security-Automation)**
-  ([ical](https://raw.githubusercontent.com/ansible/community/master/meetings/ical/Security-Automation.ics))
+  ([ical](https://raw.githubusercontent.com/ansible/community/master/meetings/ical/security.ics))
```
Comment (gundalow) [inline meetings/README.md]: Not sure if this should have changed

Comment (anshulbehl) [reply meetings/README.md]: Will send out a new commit for fixes.

**Pull request closed without merging**